<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsUi</class>
 <widget class="QWidget" name="SettingsUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset theme="preferences-desktop">
    <normaloff/>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-325</y>
        <width>451</width>
        <height>674</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupGeneral">
         <property name="title">
          <string>General Interface Settings</string>
         </property>
         <layout class="QGridLayout" name="layoutMainGuiBox">
          <item row="5" column="0">
           <widget class="QLabel" name="spacer">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="8" column="1" colspan="2">
           <widget class="QCheckBox" name="checkOsd">
            <property name="text">
             <string>Show on mouse move</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="13" column="0" colspan="3">
           <widget class="QLabel" name="labelMouseWheel">
            <property name="text">
             <string>Mouse wheel:</string>
            </property>
           </widget>
          </item>
          <item row="16" column="0" colspan="3">
           <widget class="QLabel" name="labelToolbarStyle">
            <property name="text">
             <string>Toolbar style:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1" colspan="2">
           <widget class="QCheckBox" name="checkTrayHide">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Minimise to tray</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="14" column="1" colspan="2">
           <widget class="QRadioButton" name="radioWheelChannel">
            <property name="text">
             <string>Change channel</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="15" column="1" colspan="2">
           <widget class="QRadioButton" name="radioWheelVolume">
            <property name="text">
             <string>Change volume</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QLabel" name="labelTray">
            <property name="text">
             <string>Tray icon:</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0" colspan="3">
           <widget class="QLabel" name="labelInfo">
            <property name="text">
             <string>Info panel:</string>
            </property>
           </widget>
          </item>
          <item row="17" column="1" colspan="2">
           <widget class="QComboBox" name="comboToolbarLook">
            <property name="currentIndex">
             <number>4</number>
            </property>
            <item>
             <property name="text">
              <string>Icon only</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Text only</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Text beside icon</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Text under icon</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>System default</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="7" column="0" colspan="3">
           <widget class="QLabel" name="labelOsdControls">
            <property name="text">
             <string>Controls:</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1" colspan="2">
           <widget class="QCheckBox" name="checkInfo">
            <property name="text">
             <string>Show on mouse move</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QCheckBox" name="checkRememberSize">
            <property name="text">
             <string>Remember main window size, position and open panels</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1" colspan="2">
           <widget class="QCheckBox" name="checkTray">
            <property name="text">
             <string>Show tray icon</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="12" column="1" colspan="2">
           <widget class="QCheckBox" name="checkFilter">
            <property name="text">
             <string>Filters always visible</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupIcons">
         <property name="title">
          <string>Icon theme</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QComboBox" name="comboIcons">
            <item>
             <property name="text">
              <string>Default</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Desktop theme</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&lt;i&gt;This option requires restart.&lt;/i&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="startupGuiBox">
         <property name="title">
          <string>Default options for startup</string>
         </property>
         <layout class="QVBoxLayout" name="layoutStartupGuiBox">
          <item>
           <widget class="QCheckBox" name="checkLite">
            <property name="text">
             <string>Simple mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkTop">
            <property name="text">
             <string>Always on top</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>34</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>checkRememberSize</tabstop>
  <tabstop>checkTray</tabstop>
  <tabstop>checkTrayHide</tabstop>
  <tabstop>checkOsd</tabstop>
  <tabstop>checkInfo</tabstop>
  <tabstop>checkFilter</tabstop>
  <tabstop>radioWheelChannel</tabstop>
  <tabstop>radioWheelVolume</tabstop>
  <tabstop>comboToolbarLook</tabstop>
  <tabstop>comboIcons</tabstop>
  <tabstop>checkLite</tabstop>
  <tabstop>checkTop</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkTray</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkTrayHide</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
